src/api_manager/utils/url_util.h

```diff
@@ -27,6 +28,10 @@ std::string GetUrlContent(const std::string &url);
 
 bool IsHttpRequest(const std::string &url);
 
+void Split(const std::string &s, char delim, std::vector<std::string> *elems);
+
```

Comment:
please add them to two new files  str_utils.h and str_utils.cc

Reply:
Done.